Objects/unicodeobject.c

```diff
@@ -6802,17 +6802,21 @@ _PyUnicode_DecodeUnicodeEscapeStateful(const char *s,
         unsigned char c = *first_invalid_escape;
         if ('4' <= c && c <= '7') {
             if (PyErr_WarnFormat(PyExc_DeprecationWarning, 1,
-                                 "invalid octal escape sequence '\\%.3s'",
-                                 first_invalid_escape) < 0)
+                                 "\"\\%.3s\" is an invalid octal escape sequence. "
+                                 "Such sequences will not work in the future. "
+                                 "Did you mean \"\\\\%.3s\"? A raw string is also an option.",
```

Comment:
I don't think that that the hint "Did you mean ...?" is relevant here, I suggest to remove it.

Reply:
Yeah that makes sense. Thanks for the suggestion.

Reply:
Done.